components/FlexDiv/index.js

```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components";
+
+const FlexDiv = styled.div`
```

Comment:
I think it always has to be Styled at the beginning.
also `const StyledFlexDiv = styled.div`